modular_pariah/modules/cryosleep/code/cryopod.dm

```diff
@@ -220,12 +220,13 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 				mind.announce_objectives()
 		else if(istype(objective.target) && objective.target == mob_occupant.mind)
 			if(istype(objective, /datum/objective/contract))
-				var/datum/antagonist/traitor/affected_traitor = objective.owner.has_antag_datum(/datum/antagonist/traitor)
-				var/datum/contractor_hub/affected_contractor_hub = affected_traitor.contractor_hub
-				for(var/datum/syndicate_contract/affected_contract as anything in affected_contractor_hub.assigned_contracts)
+				var/datum/contractor_hub/the_hub = GLOB.contractors[objective?.owner]
```

Comment:
This is all inside a loop, so if `objective` doesn't exist it won't ever get to this point.
https://github.com/pariahstation/Pariah-Station/blob/4e05ec6b450727c43c4bebf0df3dcfd1c43e5de9/modular_pariah/modules/cryosleep/code/cryopod.dm#L212
```suggestion
				var/datum/contractor_hub/the_hub = GLOB.contractors[objective.owner]
```